assets/js/src/core/components/tree/components/context-menu/asset-tree-context-menu.tsx

```diff
@@ -56,7 +173,36 @@ export const AssetTreeContextMenu = (props: AssetTreeContextMenuProps): React.JS
           }
         }
       ]
-    }
+    },
+    addFolder(),
+    rename({
+      onClick: () => {
+        if (props.node !== undefined) {
+          setDefaultValue({ input: props.node?.label })
+          showInputModal()
+        }
+      }
+    }),
+    copy({ nodeId: props.node?.id ?? null }),
+    paste(),
+    cut(),
+    remove({
+      onClick: () => {
+        setConfirmationText(() => (
+          <Box component={ 'center' }>
+            <span>Do you really want to delete this item?</span> <br />
```

Comment:
Please add translation keys for those messages

Reply:
https://github.com/pimcore/studio-ui-bundle/pull/564#discussion_r1794836994